### PR TITLE
docs(events): add events listener options

### PR DESCRIPTION
**Pull request by koreanddinghwan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

- too few information about event-listener options

## What is the new behavior?

event listener options object updated at [feat: add suppress errors option](https://github.com/nestjs/event-emitter/pull/936)
i think need more information to users who use events.

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/techniques/events.md [hunk 1]
```diff
@@ -94,6 +117,9 @@ handleOrderCreatedEvent(payload: OrderCreatedEvent) {
 }
 ```
 
+> info **Hint** read more about ([listener options object](https://github.com/EventEmitter2/EventEmitter2#emitteronevent-listener-options-objectboolean)).
```
[the remark this answers is not in the document]
Comment (micalevisk) [reply content/techniques/events.md:120]: also, can we move this HINT to right below L111?

[the remark this answers is not in the document]
Comment (koreanddinghwan) [reply content/techniques/events.md:120]: i think that it is better.

[the remark this answers is not in the document]
Comment (koreanddinghwan) [reply content/techniques/events.md]: ok, move to below L111

[the remark this answers is not in the document]
Comment (koreanddinghwan) [reply content/techniques/events.md]: [f61b56c](https://github.com/nestjs/docs.nestjs.com/pull/2888/commits/f61b56cf144f18ed77f9893110b7b5b4460c5a84)

**Review by micalevisk — APPROVED** [review]

**Pull request merged**